code/game/gamemodes/clock_cult/clock_effects/servant_blocker.dm

```diff
@@ -13,8 +13,9 @@
 
 /obj/effect/clockwork/servant_blocker/CanPass(atom/movable/M, turf/target)
 	var/list/target_contents = M.GetAllContents() + M
+	var/obj/structure/destructible/clockwork/massive/celestial_gateway/Ark = GLOB.ark_of_the_clockwork_justiciar
 	for(var/mob/living/L in target_contents)
-		if(is_servant_of_ratvar(L) && get_dir(M, src) != dir) //Unless we're on the side the arrow is pointing directly away from, no-go
+		if(is_servant_of_ratvar(L) && get_dir(M, src) != dir && L.stat != DEAD && Ark.progress_in_seconds < GATEWAY_REEBE_FOUND) //Unless we're on the side the arrow is pointing directly away from, no-go
```

Comment:
This doesn't check if `Ark` is null, which could runtime in the event that it was tried if the Ark was deleted or destroyed.